modules/lambdas/main.tf

```diff
@@ -479,3 +479,300 @@ data "aws_lambda_invocation" "db_migration" {
+    types = ["PRIVATE"]
+    # Cumulus might need to create vpc_endpoint_ids which should be added here in the future in order to access this API
+  }
+}
+resource "aws_api_gateway_resource" "orca_catalog_reporting_api_resource" {
+  path_part   = "{proxy+}"
+  parent_id   = aws_api_gateway_rest_api.orca_catalog_reporting_api.root_resource_id
+  rest_api_id = aws_api_gateway_rest_api.orca_catalog_reporting_api.id
+}
+
+resource "aws_api_gateway_method" "orca_catalog_reporting_api_method" {
+  rest_api_id = aws_api_gateway_rest_api.orca_catalog_reporting_api.id
+  resource_id = aws_api_gateway_resource.orca_catalog_reporting_api_resource.id
+  http_method = "ANY"
+  # todo: Make sure this is locked down against external access.
+  authorization = "NONE"
+}
+
+resource "aws_api_gateway_integration" "orca_catalog_reporting_api_integration" {
+  rest_api_id             = aws_api_gateway_rest_api.orca_catalog_reporting_api.id
+  resource_id             = aws_api_gateway_resource.orca_catalog_reporting_api_resource.id
+  http_method             = aws_api_gateway_method.orca_catalog_reporting_api_method.http_method
+  integration_http_method = "POST"
+  type                    = "AWS"
+  uri                     = aws_lambda_function.orca_catalog_reporting.invoke_arn
+}
+
+resource "aws_api_gateway_method_response" "orca_catalog_reporting_response_200" {
+  rest_api_id = aws_api_gateway_rest_api.orca_catalog_reporting_api.id
+  resource_id = aws_api_gateway_resource.orca_catalog_reporting_api_resource.id
+  http_method = aws_api_gateway_method.orca_catalog_reporting_api_method.http_method
+  status_code = "200"
+}
+
+resource "aws_api_gateway_integration_response" "orca_catalog_reporting_api_response" {
+  depends_on  = [aws_api_gateway_integration.orca_catalog_reporting_api_integration]
+  rest_api_id = aws_api_gateway_rest_api.orca_catalog_reporting_api.id
+  resource_id = aws_api_gateway_resource.orca_catalog_reporting_api_resource.id
+  http_method = aws_api_gateway_method.orca_catalog_reporting_api_method.http_method
+  status_code = aws_api_gateway_method_response.orca_catalog_reporting_response_200.status_code
+
+  # Transforms the backend JSON response to XML
+  response_templates = {
+    "application/json" = <<EOF
+    #set($inputRoot = $input.path('$'))
+    $input.json("$")
+    #if($input.path("stackTrace") != '')
+    #set($context.responseOverride.status = 500)
+    #elseif($input.path("httpStatus") != '')
+    #set($context.responseOverride.status = $input.path('httpStatus'))
+    #end
+    EOF
+  }
+}
+
+resource "aws_api_gateway_deployment" "orca_catalog_reporting_api_deployment" {
+  rest_api_id = aws_api_gateway_rest_api.orca_catalog_reporting_api.id
+}
+
+resource "aws_api_gateway_stage" "orca_catalog_reporting_api_stage" {
+  deployment_id = aws_api_gateway_deployment.orca_catalog_reporting_api_deployment.id
+  rest_api_id   = aws_api_gateway_rest_api.orca_catalog_reporting_api.id
+  stage_name    = "dev"
```

Comment:
I wonder if this should not be changed "dev" implies development. This should probably be something like orca or orca_production or something like that to denote that it would be a production ready stage.

Reply:
I can change this to `orca`